idioms/default.md

```diff
@@ -0,0 +1,50 @@
+# The `Default` Trait
+
+## Description
+
+Many types in Rust have a [Constructor]. However, this is *specific* to the
+type; Rust cannot abstract over "everything that has a `new()` method". To
+allow this, the [`Default`] trait was conceived, which can be used with
+containers and other generic types (e.g. see [`Option::unwrap_or_default()`]).
+Notably, some containers already implement it where applicable.
```

Comment:
Worth noting, though probably further down, is that `new` and `default` do different things - `new` typically takes values for all or most fields (i.e., is shorthand for the literal, modulo private fields), whereas `default` takes no arguments and thus creates an object 'from nothing'.